src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -152,12 +165,16 @@
     <UpdateNuGetConfigPackageSourcesMappings
       NuGetConfigFile="$(NuGetConfigFile)"
       BuildWithOnlineFeeds="$(DotNetBuildWithOnlineFeeds)"
-      SourceBuildSources="$(SourceBuildSources)" />
+      SourceBuildSources="@(SourceBuildSources)" />
 
     <MakeDir Directories="$(BaseIntermediateOutputPath)" />
     <Touch Files="$(BaseIntermediateOutputPath)UpdateNuGetConfig.complete" AlwaysCreate="true">
       <Output TaskParameter="TouchedFiles" ItemName="FileWrites" />
     </Touch>
+
+    <ItemGroup>
+      <EmbedInBinlog Include="$(NuGetConfigFile)" />
+  </ItemGroup>
```

Comment:
Move this into repo-projects/Directory.Build.props. I don't think this should be in a target.

Reply:
It needs to be in a target because we want the NuGet.config write operations done before they get embedded. Otherwise, we end up with the original version of the NuGet.config from the repo's source.